int32 wr_sec
int32 wr_nsec
int32 rd_sec
int32 rd_nsec
float64 a_x
float64 a_y
float64 v_z
float64 yaw_rate